### PR TITLE
Add file storage cache

### PR DESCRIPTION
**Pull request by dursk**
Writing json to the file because it's just easier to deal with in python than doing some other weird shit and string splitting, etc etc.

### CODE REVIEW

File: mbq/client/storage.py [hunk 1]
```diff
@@ -11,3 +12,28 @@ def set(self, key, value):
...
+
+class FileStorage(object):
+
+    def __init__(self, filename):
+        self.filename = filename
+
+    def set(self, key, value):
+        with open(self.filename, 'r') as f:
+            try:
+                tokens = json.loads(f.read())
```
Comment (mattbriancon) [inline mbq/client/storage.py]: nit: `json.load(f)` and `json.dump(f)` work too.

File: mbq/client/tests/compat.py [hunk 2]
```diff
@@ -1,8 +1,4 @@
 try:
-    from unittest import mock
+    from unittest.mock import *  # noqa
```
Comment (mattbriancon) [inline mbq/client/tests/compat.py]: IMO we should just expose `mock` not all (or any) of it's members.

File: mbq/client/tests/test_storage.py [hunk 3]
```diff
@@ -0,0 +1,53 @@
+from unittest import TestCase
+
+from .compat import mock_open, patch
+
+from mbq.client.storage import FileStorage
+
+try:
+    from io import StringIO
+except ImportError:
+    from stringIO import StringIO  # noqa
```
Comment (mattbriancon) [inline mbq/client/tests/test_storage.py]: Can we move this to compat.py?

File: mbq/client/tests/compat.py (hunk 2 above)
Comment (dursk) [reply mbq/client/tests/compat.py]: Why not? `compat` is now just the same thing as `mock`

Comment (dursk) [reply mbq/client/tests/compat.py]: Ah, nvm, I just saw your other comment

**Pull request merged**